python/functional-programming/generators/benefits-of-using-generators.md

```diff
@@ -40,6 +40,14 @@ Generators are functions which behave like iterators, but can have better perfor
 
 However, the set of generated values can only be used once.
 
+---
+## Practice
+
+When using generators ??? wait for all the values to be generated before using them.
```

Comment:
I think this one needs a code sample, and a pythontutor link to really explain it